app/components/works/publication_date_component.html.erb

```diff
@@ -21,6 +18,11 @@
       <label for="work_published_day" class="visually-hidden">Publication day</label>
       <%= day_field %>
     </div>
+
+    <%= button_tag type: 'button', class: 'btn btn-sm ms-3', aria: { label: "Clear" },
```

Comment:
Can this label be more descriptive than "Clear"? "Clear published date" perhaps?

Reply:
Yes. Done.